app/src/main/java/mozilla/lockbox/view/ItemViewHolder.kt

```diff
@@ -18,7 +19,15 @@ class ItemViewHolder(override val containerView: View) : ItemListCell(containerV
             field = value
             value?.let {
                 itemTitle.text = it.title
-                itemSubtitle.text = it.subtitle
+                itemSubtitle.text = setSubtitlePlaceholderText(it.subtitle)
             }
         }
+
+    private fun setSubtitlePlaceholderText(subtitle: String?): CharSequence {
```

Comment:
this method name is a little awkward... it doesn't actually `set` anything. I might call it `textFromSubtitle(subtitle: String?)` or something similar....